meta-ov/recipes-apps/ovmenu-ng-skripts/files/download-all.sh

```diff
@@ -1,19 +1,18 @@
 #!/bin/bash
 
-USB_PATH="/usb/usbstick/openvario/download/xcsoar"
+USB_PATH="/usb/usbstick/openvario/download/xcsoar/"
 DOWNLOAD_PATH="/home/root/.xcsoar"
 
-if [ ! -d "$USB_PATH" ]; then
-	mkdir -p "$USB_PATH"
+if [ ! -d "${USB_PATH}" ]; then
+	mkdir -p "${USB_PATH}"
 fi
 
-if [ -z "$(ls $DOWNLOAD_PATH/* 2>/dev/null)" ]; then
+if [ -z "$(ls ${DOWNLOAD_PATH}/* 2>/dev/null)" ]; then
```

Comment:
And that? No explanation in the commit message what exactly is being fixed ...

Reply:
curly braces are best practice.